spec/factories/ingridient.rb

```diff
@@ -0,0 +1,50 @@
+FactoryBot.define do
+  factory :ingridient do
+
```

Comment:
✂️

Reply:
`This is also reported by CC -> Extra empty line detected at block body beginning.`